presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSourceFactoryUtils.java

```diff
@@ -111,14 +112,14 @@ public static OrcReader getOrcReader(OrcEncoding orcEncoding, List<HiveColumnHan
     public static PrestoException mapToPrestoException(Exception e, Path path, HiveFileSplit fileSplit)
     {
         // instanceof and class comparison do not work here since they are loaded by different class loaders.
```

Comment:
Isn't this comment still relevant

Reply:
@tdcmeehan Shall I replace all occurrences of "instanceof" with "isAssignableFrom" in the mapToPrestoException method?

Reply:
I'm just thinking that the problem is still present, because you're comparing potentially the same class but with two different classloaders.

Reply:
1. isAssignableFrom won't work for classes loaded with different classloaders. 
2. I don't see any security threats posed by this class name comparison logic. I'd discard this PR.

Reply:
@tdcmeehan @sdruzkin Ok. Then i will mark these issues as false positive and close this PR.